libbeat/docs/outputconfig.asciidoc

```diff
@@ -86,13 +86,85 @@ output.elasticsearch:
   protocol: "https"
   username: "{beatname_lc}_internal"
   password: "{pwd}"
-
 ------------------------------------------------------------------------------
 
 
 For more information about securing {beatname_uc}, see
 <<securing-{beatname_lc}>>.
 
+
+To configure the Elasticsearch output with https://www.elastic.co/guide/en/elasticsearch/reference/6.x/getting-started-index-lifecycle-management.html[index lifecycle management] feature configure it as following:
```

Comment:
@dedemorton I'm not very happy with the docs yet. I'm thinking to take them out of this PR and then we sit together to discuss how we should structure this docs and where they should go? Would appreciate your help on this one.